### PR TITLE
Update ru.po

### PR DESCRIPTION
**Pull request by Alsirion**
Changed translate words Mount and Volume.

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -1387,7 +1387,7 @@ msgstr "Перевозчик"
 
 #: pkg/kubernetes/scripts/volumes.js:205
 msgid "Ceph Filesystem Mount"
-msgstr "Крепление файловой системы Ceph"
+msgstr "Монтирование файловой системы Ceph"
```
Comment (martinpitt) [inline po/ru.po]: Reference: https://ru.wikipedia.org/wiki/Mount uses that word as well, although it's not on the [leo dict](https://dict.leo.org/russisch-deutsch/%D0%9C%D0%BE%D0%BD%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5).

File: po/ru.po [hunk 2]
```diff
@@ -3356,7 +3356,7 @@ msgstr "Группы"
 #: pkg/storaged/lvol-tabs.jsx:395 pkg/storaged/vdo-details.jsx:218
 #: pkg/storaged/vdo-details.jsx:293
 msgid "Grow"
-msgstr "расти"
+msgstr "Увеличить"
```
Comment (martinpitt) [inline po/ru.po]: these are both in the dictionary and seem to mean the same thing by and large? Is Увеличить more common in computer language?

File: po/ru.po [hunk 3]
```diff
@@ -5463,7 +5463,7 @@ msgstr "раздел"
 
 #: pkg/storaged/utils.js:263
 msgid "Partition of $0"
-msgstr "Разделение $0"
+msgstr "Раздел $0"
```
Comment (martinpitt) [inline po/ru.po]: ack, reference: https://ru.wikipedia.org/wiki/%D0%A0%D0%B0%D0%B7%D0%B4%D0%B5%D0%BB_%D0%B4%D0%B8%D1%81%D0%BA%D0%B0

File: po/ru.po [hunk 4]
```diff
@@ -9076,7 +9076,7 @@ msgstr "Доступна служба виртуализации"
 #: pkg/machines/components/vmDiskSourceCell.jsx:41
 #: pkg/machines/components/diskAdd.jsx:91 pkg/storaged/content-views.jsx:127
 msgid "Volume"
-msgstr "Громкость"
+msgstr "Том"
```
Comment (martinpitt) [inline po/ru.po]: Indeed that sounds more sensible :)

File: po/ru.po (hunk 2 above)
Comment (martinpitt) [reply po/ru.po]: I'm not applying that yet, as there are similar strings which also use расти. E. g:

- Grow logical size of $0 → Растите логический размер $0
- Grow to take all space → Растите все пространство

**Comment by martinpitt** [comment]
The changed translations are part of PR #10936

File: po/ru.po (hunk 2 above)
Comment (Alsirion) [reply po/ru.po]: Grow - used in UI near with string 7182. And string 7182 translated as "Уменьшить" and normal use antonym as "Увеличить". But "Расти" - it's understandable.

And i saw near mistake in string 7186 its not normal translate, but i don't saw this in real UI and can't comment this string - but it's wrong translate. It's absolutely  not "Термоусадочный"

Comment (martinpitt) [reply po/ru.po]: I'm afraid I don't understand what should happen here -- should "Grow" *and* the other related  strings be changed consistently? Or keep расти for now? Not sure what "7812" refers  to, apparently not a line number in ru.po?

Comment (Alsirion) [reply po/ru.po]: I'm not linguist or translator, I'm only cockpit user, but Grow as Расти - it's clear for understand and may stay this translate. It's similar meanings and if change one of them, then must change all other.

Comment (martinpitt) [reply po/ru.po]: Thanks for the follow-up! Let's close this then, as all the other changes got applied.

**Pull request closed without merging**